command/ca/provisioner/provisioner.go

```diff
@@ -314,21 +314,61 @@ disable ssh use '--ssh=false'.`,
 containing one or more PEM formatted keys, if used with the K8SSA provisioner.`,
 	}
 
+	// ACME and SCEP provisioner flags
 	forceCNFlag = cli.BoolFlag{
 		Name:  "force-cn",
 		Usage: `Always set the common name in provisioned certificates.`,
 	}
-	requireEABFlag = cli.BoolFlag{
+
+	challengeFlag = cli.StringSliceFlag{
+		Name: "challenge",
+		Usage: `With a SCEP provisioner is the <challenge> to use as a shared secret between a
```

Comment:
With a SCEP provisioner the <challenge> is a shared secret between the client and the CA.

Reply:
Yes

Reply:
@dopey Should we rename `--remove-acme-challenge` to `--remove-challenge`, for consistency with the `--challenge` flag, even that doesn't make sense for scep?

Reply:
Yes, I think we should. Unless you want to add `add-acme-challenge`.

Reply:
Also, I meant that we should update the sentence in the line I linked. The grammar is weird.

Reply:
Fixed with f3bcd8a